### ISSUE TITLE
freeze reaction on anti-freeze bosses

### ISSUE THREAD

**Issue by leepc12** [genshinsim/gcsim#1322]
I'd like to simulate Ayaka freeze comp against anti-freeze bosses.

I can work around it by setting 2p-Blizzard Strayer on Ayaka and give her extra 20% crit rate stat.

```
#ayaka add set="blizzardstrayer" count=4;
ayaka add set="blizzardstrayer" count=2;
ayaka add stats cr=0.2;
```

This assumes that cryo aura is 100% uptime, which is not the case since every hydro application of Kokomi E removes anti-freeze mob's cryo status.

I'm still looking for more accurate way to implement it. For example, forcing zero freeze duration. Any good idea on this?

**Comment by unleashurgeek** [comment]
This is currently not implemented. This open PR adds the ability to have freeze immune targets, but no eta on when it will be merged in: https://github.com/genshinsim/gcsim/pull/1105

**Comment by k0l11** [comment]
should be resolved by #1335. reopen if it isn't
